ApplyToBecomeInternal/ApplyToBecomeInternal/Models/NotificationType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace ApplyToBecomeInternal.Models
+{
+	public enum NotificationType
+	{
+		Success,
```

Comment:
Enum should ideally be assigned an explicit number for each value.

Reply:
Sure, will address that now.

Reply:
This has been resolved in the latest commit.